algebird-core/src/main/scala/com/twitter/algebird/QTree.scala

```diff
@@ -258,6 +289,7 @@ case class QTree[A](
   }
 
   def interQuartileMean(implicit n: Numeric[A]): (Double, Double) = {
+    implicit val monoid: Monoid[A] = Ring.numericRing[A]
```

Comment:
That seems like a pretty decent one, odds of the Numeric being incompatible in its toDouble conversion seem low